tests/tools/transpilers/test_python.py

```diff
@@ -766,3 +766,29 @@ def test_transpiler_literal_datetime() -> None:
     assert (
         generated_code == expected_code
     ), f"Expected '{expected_code}', but got '{generated_code}'"
+
+
+###->
+def test_transpiler_classdefstmt() -> None:
+    """Test astx.ClassDefStmt."""
+    # Class body
+    class_body = astx.Block(name="MyClass_body")
+    var1 = astx.Variable(name="var1")
+    class_body.append(var1)
+
+    # Create a class definition
+    class_def = astx.ClassDefStmt(
+        name="MyClass",
+        body=class_body,
+    )
+
+    # Initialize the generator
+    generator = astx2py.ASTxPythonTranspiler()
```

Comment:
you don't need to instantiate the transpiler anymore .. you can use it from here: https://github.com/arxlang/astx/pull/153/files#diff-9a73ce80bf21a791d186ea32cb2cebc60eface5143e46a9c27f367a8bd04d98dL7